src/timescale/commit_comment_github_events.service.ts

```diff
@@ -42,6 +43,10 @@ export class CommitCommentGithubEventsService {
       .where(`LOWER(actor_login) = '${username}'`)
       .groupBy("LOWER(actor_login)");
 
+    if (repos && repos.length > 0) {
+      queryBuilder.andWhere(`repo_name IN (:...repos)`, { repos: repos.map((repo) => repo.toLowerCase()) });
```

Comment:
This won't hit the index since we've engaged the index on `LOWER(repo_name)`. And you need to provide a comma delimited list to the sql. So, you'll want to change:

```suggestion
      queryBuilder.andWhere(`LOWER(repo_name) IN (:...repos)`, { repos: repos.map((repo) => repo.toLowerCase()) });
```

and find a way to stitch together all the repos to one string delimited by a comma.

Example:

https://github.com/open-sauced/api/blob/57542a7570d59b1233805a3d50e6ec51a6daa455/src/timescale/watch_github_events.service.ts#L62-L67

Reply:
AFAIK, these two are equivalent, as the parameters object is the second argument.

```ts
 if (options.repo) { 
   queryBuilder.andWhere(`LOWER("watch_github_events"."repo_name") IN (:...repoNames)`, { 
     repoNames: options.repo.toLowerCase().split(","), 
   }); 
 } 
```
```ts
 if (options.repo) { 
   queryBuilder.andWhere(`LOWER("watch_github_events"."repo_name") IN (:...repoNames)`).setParameters({ 
     repoNames: options.repo.toLowerCase().split(","), 
   }); 
 } 
```

I'm fine with going with the `setParameters` approach if that's preferred.

Reply:
Oh yeah, either or is totally fine: just needs the `LOWER` and the comma delimited string. But you can do it either way, I don't think there's much of a difference. Not sure why I went with `setParameters` in the past beyond seeing it in some nest examples